packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/utils/tableHeaders.js

```diff
@@ -16,18 +16,18 @@ const tableHeaders = [
     },
   },
   {
-    name: 'type',
-    key: 'type',
+    name: 'createdAt',
+    key: 'createdAt',
     metadatas: {
-      label: 'Token type',
+      label: 'Created at',
       sortable: false,
     },
   },
   {
-    name: 'createdAt',
-    key: 'createdAt',
+    name: 'lastUsedAt',
+    key: 'lastUsedAt',
     metadatas: {
-      label: 'Created at',
+      label: 'Last used',
```

Comment:
@ronronscelestes Will merge this PR today which conflicts with these changes: https://github.com/strapi/strapi/pull/14111 and after this branch needs an update + merge. Could you have a look and take care of that?

Reply:
ok sure

Reply:
done